scss/app/demo.scss

```diff
@@ -191,6 +191,14 @@
     background: $black87;
   }
 
+  .buttons-group--block {
+    padding: 3px;
+
+    button {
```

Comment:
I just override the parent style: https://github.com/jusbrasil/monalisa/pull/22#discussion-diff-24467593R184

I will refactor the both